src/Microsoft.Health.Dicom.Functions.UnitTests/CleanupDeleted/CleanupDeletedDurableFunctionTests.Activity.cs

```diff
@@ -0,0 +1,123 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Health.Dicom.Core.Features.ChangeFeed;
+using Microsoft.Health.Dicom.Core.Features.Model;
+using Microsoft.Health.Dicom.Functions.BlobMigration.Models;
+using Microsoft.Health.Dicom.Tests.Common;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Functions.UnitTests.CleanupDeleted;
+
+public partial class CleanupDeletedDurableFunctionTests
+{
+    [Fact]
+    public async Task GivenTimStamp_WhenGettingDeletedChangeFeedInstanceBatches_ThenShouldInvokeCorrectMethod()
+    {
+        const int batchSize = 100;
+        DateTime timeStamp = DateTime.UtcNow;
+
+        IReadOnlyCollection<ChangeFeedEntry> expected = new List<ChangeFeedEntry> {
+            new ChangeFeedEntry(1, DateTime.Now, ChangeFeedAction.Delete, TestUidGenerator.Generate(), TestUidGenerator.Generate(), TestUidGenerator.Generate(), 1, 1, ChangeFeedState.Current),
+            new ChangeFeedEntry(2, DateTime.Now, ChangeFeedAction.Delete, TestUidGenerator.Generate(), TestUidGenerator.Generate(), TestUidGenerator.Generate(), 2, 2, ChangeFeedState.Current),
+        };
+
+        _changeFeedStore
+            .GetDeletedChangeFeedByWatermarkOrTimeStampAsync(batchSize, timeStamp)
```

Comment:
I can't seem to find this method haha

Reply:
I didnt merge my sql PR, merged now. Its in main